docs/user-guide/text-editor-toolbar.md

```diff
@@ -1,27 +1,29 @@
 # Text Editor Toolbar
 *********************
 
-With the *text editor toolbar* it is possible to customize the text by modifying the following aspects:
+With the *Text Editor Toolbar* it is possible to customize the text by modifying the following aspects:
 
 <img src="../img/text-editor-toolbar/text-editor-toolbar.jpg" class="ms-docimage"/>
 
-* **Block Type** to change the text size (*Small*, *Normal*, *Large*, *Huge*)
+* **Font** to choose the text font (*Inherit*, *Arial*, *Georgia*, *Impact*, *Tahoma*, *Time New Roman*, *Verdana*)
 
-<img src="../img/text-editor-toolbar/block-type.jpg" class="ms-docimage" style="max-width:150px;"/>
+* **Block Type** to change the text size (*Normal*, *H1*, *H2*, *H3*, *H4*, *H5*, *H6*, *Blockquote*, *Code*)
```

Comment:
* The **Block Type** by choosing between the available ones (*Normal*, *H1*, *H2*, *H3*, *H4*, *H5*, *H6*, *Blockquote*, *Code*)